coral/src/app/main.tsx

```diff
@@ -6,12 +6,25 @@ import "@aivenio/design-system/dist/styles.css";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
 import "/src/app/main.module.css";
+import { isUnauthorizedError } from "src/services/api";
 
 const DEV_MODE = import.meta.env.DEV;
 
 const root = createRoot(document.getElementById("root") as HTMLElement);
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      useErrorBoundary: (error: unknown) => isUnauthorizedError(error),
```

Comment:
Shouldn't this also be added for `mutations` ?

Reply:
We are just building the first page, which will only display data, so I would revisit the whole mutations defaults when we introduce the first one.